syft/pkg/cataloger/cpp/package.go

```diff
@@ -8,23 +8,67 @@ import (
 	"github.com/anchore/syft/syft/pkg"
 )
 
-func newConanfilePackage(m pkg.ConanMetadata, locations ...file.Location) *pkg.Package {
-	fields := strings.Split(strings.TrimSpace(m.Ref), "/")
-	if len(fields) < 2 {
-		return nil
+type ConanRef struct {
```

Comment:
nit: this does not need to be exported, since it's only used in this package. `ConanRef` -> `conanRef`

Reply:
I went ahead and pushed this change